src/middleware.js

```diff
@@ -157,6 +157,29 @@ export function withDagula (handler) {
   }
 }
 
+/**
+ * Intercepts request if content cached by just returning cached response.
+ * Otherwise proceeds to handler.
+ * @type {Middleware}
+ */
+export function withCdnGet (handler) {
```

Comment:
Maybe, `withCdnCache` to avoid confusion with `withHttpGet` (which restricts to HTTP `GET` method)?